code/game/objects/items/devices/lightreplacer.dm

```diff
@@ -116,15 +116,16 @@
 			return
 
 /obj/item/device/lightreplacer/attack_self(mob/user)
-	/* // This would probably be a bit OP. If you want it though, uncomment the code.
-	if(isrobot(user))
-		var/mob/living/silicon/robot/R = user
-		if(R.emagged)
-			src.Emag()
-			to_chat(usr, "You shortcircuit the [src].")
-			return
-	*/
-	to_chat(usr, "It has [uses] lights remaining.")
```

Comment:
?

Reply:
The user can see the remaining lights by examining the tool.

Reply:
> The user can see the remaining lights by examining the tool.

oh this is fine then